### PR TITLE
804 update dropdown pattern

### PR DESCRIPTION
**Pull request by willdoran**
Changing drop down menus to match platform pattern library

[Review on Reviewable](https://reviewable.io/reviews/ushahidi/platform-client/36)

### CODE REVIEW

File: server/www/templates/posts/post-view-filters.html [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <div class="filters" dropdown auto-close="disabled">
-    <div class="filters-show-hide" ng-class="{ half : halfWidth }">
+    <div class="filters-toggle init" ng-if="! hasAside" ng-class="{ half : halfWidth }" dropdown-toggle>
```
Comment (rjmackay) [inline server/www/templates/posts/post-view-filters.html]: careful with ng-if .. it changes the scope of everything inside this element.

File: app/post/directives/post-view-filters-directive.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ function (
         replace: true,
         scope: {
             halfWidth: '=',
+            hasAside: '=',
```
Comment (rjmackay) [inline app/post/directives/post-view-filters-directive.js]: Can you double check hasAside isn't just a dupe of halfWidth?

**Comment by rjmackay** [comment]
Notice this doesn't change the classes used in the dropdown directive at all?? I thought the PL changes had change the names of the open/closed classes?

File: server/www/templates/posts/post-view-filters.html (hunk 1 above)
Comment (willdoran) [inline server/www/templates/posts/post-view-filters.html]: that's gross - I'll change it to a show!

File: app/post/directives/post-view-filters-directive.js (hunk 2 above)
Comment (willdoran) [inline app/post/directives/post-view-filters-directive.js]: Half width does appear to be redundant so I've removed it!

**Comment by willdoran** [comment]
The main changes were order and structural. The class names themselves didn't change but they did move around a bit. The inclusion of the init and active elements for drop downs and then the changes to how filters are order hierarchically.

**Pull request merged**